src/Symfony/Component/ClassLoader/ClassCollectionLoader.php

```diff
@@ -90,12 +90,37 @@ public static function load($classes, $cacheDir, $name, $autoReload, $adaptive =
             return;
         }
 
+        self::inline($classes, $cache, $declared);
+
+        if ($autoReload) {
+            // save the resources
+            self::writeCacheFile($metadata, serialize(array($files, $classes)));
```

Comment:
but then, should ``inline`` really write the file if it does not write the meta file ?